app/api/api/healthcheck.py

```diff
@@ -7,8 +7,8 @@
 from werkzeug.exceptions import ServiceUnavailable
 
 import api.adapters.db.flask_db as flask_db
-from api.route import response
-from api.route.schemas import request_schema
+from api.api import response
```

Comment:
Yeah I'm cool with `src`. At first I considered naming it `app`, but then it is confusing and breaks some files since in some files `app` is used for the local variable `Flask` app. I can go ahead and rename it to `src` unless you have any objections or other names you prefer.

Reply:
`src` works. `app` would lead to `app.app` in the path which would be its own issue (might be worth renaming that top-level folder)

Reply:
Ok renamed to `src`. Big change, hope i didn't break anything 🤞

Reply:
Reverted the change and will do it in a subsequent PR. Created ticket #144